indico/modules/events/registration/client/js/form_setup/ItemSettingsModal.jsx

```diff
@@ -153,6 +155,32 @@ export default function ItemSettingsModal({id, sectionId, defaultNewItemType, on
           )}
           {SettingsComponent && <SettingsComponent {...itemData} />}
           {renderPluginComponents(`regform-${inputType}-field-settings`, {...itemData})}
+          <Fieldset legend={Translate.string('Show if')}>
+            <FinalDropdown
+              name="showIfFieldId"
+              label={Translate.string('Field')}
+              placeholder={Translate.string('Select field...')}
+              options={itemsForConditionalDisplay.map(({title, id: fieldId}) => ({
+                value: fieldId,
+                text: title,
+              }))}
+              closeOnChange
+              selection
+            />
+            {/* TODO: Display only if showIfFieldId is not empty */}
+            <FinalDropdown
+              name="showIfFieldValue"
+              label={Translate.string('Has value')}
+              placeholder={Translate.string('Select value...')}
+              // TODO: Get possible values from the field selected in showIfFieldId
+              options={[
+                {value: '1', text: Translate.string('Yes')},
+                {value: '0', text: Translate.string('No')},
+              ]}
+              closeOnChange
+              selection
+            />
```

Comment:
I think having a single custom field that handles both parts is easier, especially since the second fields depends on the type of field selected in the first one.